commands/host.vegman

```diff
@@ -158,25 +158,70 @@ function cmd_virtualmedia {
   subcommand "$@"
 }
 
+# @sudo cmd_virtualmedia_status admin
+# @restrict cmd_virtualmedia_status admin
+# @doc cmd_virtualmedia_status
+# Get status of the virtual media
+function cmd_virtualmedia_status {
+  local usb_status_metadata=( "USB0;Proxy" "USB1;Legacy" )
+
+  for usb_item_metadata in "${usb_status_metadata[@]}"
+  do
+    local id="${usb_item_metadata%%;*}"
+    local vm_type="${usb_item_metadata#*;}"
+    local usb_status=$(busctl get-property xyz.openbmc_project.VirtualMedia \
+                                        /xyz/openbmc_project/VirtualMedia/${vm_type}/${id} \
+                                        xyz.openbmc_project.VirtualMedia.Process Active)
+    local image=""
+    echo -n "The (USB0) Virtual Media status: "
```

Comment:
- `The` seems to me excess here.
 - `USB0` looks like a missed hadrcode
 - If the tab character is really required as a delimiter put it in this statement.

So, I propose to change this line to something like:
```bash
echo -ne "Virtual Media (${id}) status:\t"
```

Reply:
fixed